templates/global/filter_form.html

```diff
@@ -1,52 +1,48 @@
 
-    <form
-            method="post"
-            id="HouseFilter"
-            novalidate
-    >
-        <table>
-            {{ form.media.js }}
-            {% for field in form %}
-                <tr>
-                    {%if not "income" in field.label_tag%}
-                        <th>{{ field.label_tag }}</th>
-                        <td colspan="2">{{ field }}</td>
-                    {% endif %}
-                </tr>
-            {% endfor %}
-            <tr style = "border:none">
-                <th colspan="3"><label>Income:</label></th>
-            </tr>
+<form method="post" id="HouseFilter" novalidate>
+    <table>
+        {{ form.media.js }}
+        {% for field in form %}
+        <tr>
+            {%if not "income" in field.label_tag%}
+            <th>{{ field.label_tag }}</th>
+            <td colspan="2">{{ field }}</td>
+            {% endif %}
+        </tr>
+        {% endfor %}
+        <tr style="border:none">
+            <th colspan="3"><label>Income:</label></th>
+        </tr>
 
-            <tr style = "border:none">
-                <td style="width:45%;padding:0px;">
-                    {{form.lowest_income.errors}}
-                    {{form.lowest_income}}
-                </td>
-                <td style="width:0;padding:0px;">
-                    <label style="font-size:1.9rem"> - </label>
-                </td>
-                 <td style="width:45%;padding:0px;">
-                     {{form.highest_income.errors}}
-                     {{form.highest_income}}
-                </td>
-            </tr>
-            <tr style = "border:none;">
-                <th style="padding:0px">
-                    <label>min</label>
-                </th>
-                 <th colspan="2" style="padding:0px">
-                    <label>max</label>
-                </th>
-            </tr>
-            <tr>
-                <td>
-                    <button type="submit">Submit</button>
-                </td>
-            </tr>
-        </table>
-        {% csrf_token %}
-    </form>
-{% endblock %}
+        <tr style="border:none">
+            <td style="width:45%;padding:0px;">
+                {{form.lowest_income.errors}}
+                {{form.lowest_income}}
+            </td>
+            <td style="width:0;padding:0px;">
+                <label style="font-size:1.9rem"> - </label>
+            </td>
+            <td style="width:45%;padding:0px;">
+                {{form.highest_income.errors}}
+                {{form.highest_income}}
+            </td>
+        </tr>
+        <tr style="border:none;">
+            <th style="padding:0px">
+                <label>min</label>
+            </th>
+            <th colspan="2" style="padding:0px">
+                <label>max</label>
+            </th>
+        </tr>
+        <tr>
+            <td>
+                <button type="submit">Submit</button>
+            </td>
+        </tr>
+    </table>
+    {% csrf_token %}
+</form>
+{% endblock %}
```

Comment:
missing line

Reply:
In html files the formating is without a new line I think